config/log4j2.properties

```diff
@@ -57,12 +57,14 @@ logger.stats.appenderRef.rolling.ref = StatsLog
 #logger.spring.name=org.springframework.jdbc
 #logger.spring.level=debug
 #logger.spring.appenderRef.rolling.ref = main
-#
-## set to DEBUG to log hsqldb SQL statement and execution
-#logger.hsqldb.name=org.hsqldb.jdbc
-#logger.hsqldb.level=debug
-#logger.hsqldb.appenderRef.rolling.ref = main
-#
+
+## set to WARN to lower the amount of logs coming from hsqldb
+logger.hsqldb.name=hsqldb.db
+logger.hsqldb.level=warn
```

Comment:
I agree with what you said.  Fine tuning logs has been an item I wanted to do, but have not since it would touch a lot of files.  I still plan to, maybe in between one of the releases. 

However, this one about hsqldb is different.  It was at DEBUG before and the only logs I've seen from them are these on every table load.  It has absolutely no value to us.
```
01/13 17:01:50  INFO - Checkpoint start [pool-5-thread-1] (HSQLDB85AC149F20.ENGINE#privlog:-1)
01/13 17:01:50  INFO - checkpointClose start [pool-5-thread-1] (HSQLDB85AC149F20.ENGINE#privlog:-1)
01/13 17:01:50  INFO - checkpointClose synched [pool-5-thread-1] (HSQLDB85AC149F20.ENGINE#privlog:-1)
01/13 17:01:50  INFO - checkpointClose script done [pool-5-thread-1] (HSQLDB85AC149F20.ENGINE#privlog:-1)
01/13 17:01:50  INFO - checkpointClose end [pool-5-thread-1] (HSQLDB85AC149F20.ENGINE#privlog:-1)
01/13 17:01:50  INFO - Checkpoint end - txts: 1 [pool-5-thread-1] (HSQLDB85AC149F20.ENGINE#privlog:-1)
```

Reply:
oh, I did not understand, the was just for `hsqldb`. Yes thats fine. Sorry.